README.md

```diff
@@ -1,107 +1,155 @@
-<h1 align="center">Invidious</h1>
+<div align="center">
+  <img src="assets/android-chrome-192x192.png">
+  <h1>Invidious</h1>
+
+  <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">
+    <img alt="License: AGPLv3+" src="https://shields.io/badge/License-AGPL%20v3+-blue.svg">
+  </a>
+  <a href="https://github.com/iv-org/invidious/actions">
+    <img alt="Build Status" src="https://github.com/iv-org/invidious/workflows/Invidious%20CI/badge.svg">
+  </a>
+  <a href="https://github.com/iv-org/invidious/issues">
+    <img alt="GitHub issues" src="https://img.shields.io/github/issues/iv-org/invidious?color=important">
+  </a>
+  <a href="https://github.com/iv-org/invidious/pulls">
+    <img alt="GitHub pull requests" src="https://img.shields.io/github/issues-pr/iv-org/invidious?color=blueviolet">
+  </a>
+  <a href="https://hosted.weblate.org/engage/invidious/">
+    <img alt="Translation Status" src="https://hosted.weblate.org/widgets/invidious/-/translations/svg-badge.svg">
+  </a>
+  <a href="https://github.com/humanetech-community/awesome-humane-tech">
```

Comment:
Maybe remove the humane tech badge? There was some controversy* regarding it within the Invidious community a couple months back. See #1331 and #1648.

*It could all be due to a misunderstanding but I personally don't have the time to delve into this.

Reply:
Most of the projects listed at [humanetech-community/awesome-humane-tech](https://github.com/humanetech-community/awesome-humane-tech) are privacy-friendly things, so I don't see how this is controversial. It's kinda the same thing as recommending extensions in the docs.

@hackerncoder @TheFrenchGhosty @Perflyst @B0pol @cloudrac3r @saltycrys @unixfox What's your opinion on this?

Reply:
I personally don't really have an opinion on this. I was mostly just pointing out the controversy regarding it a couple months ago.

Also you may want to change the 😐 emoji to something else. There's no way to add that as a reaction on GitHub.

Reply:
Humane tech's listing page is good. Bibliogram has taken their badge. Now please stop tagging me in every fucking thread, lest I silence all notifications entirely.

Reply:
> Also you may want to change the 😐 emoji to something else. There's no way to add that as a reaction on GitHub.

it's the bottom left one: 
![image](https://user-images.githubusercontent.com/52980486/126040507-ddc41124-bb92-49c0-a2cd-6cfa5bdfe5ac.png)

> Now please stop tagging me in every fucking thread, lest I silence all notifications entirely.

Sure.

Reply:
> *It could all be due to a misunderstanding from what I'm seeing but I don't have the time to delve more into this.

humanetech.community still uses resources from humanetech.com (which is the problematic org) but according to the creator of humanetech.community their project is independent. I would say we can keep it for now and if there are any complains we can re-consider.

Reply:
Going to mark this comment as off topic but I personally don't have the 😐 emoji.

![Screenshot_20210717-153341_1](https://user-images.githubusercontent.com/70992037/126050735-7910e4e8-33ab-43bc-a9d7-f40e9122fcf4.png)

Reply:
>What's your opinion on this? (if you won't comment and just react: +1 to keep it, -1 to remove it, neutral_face if you don't care)

Same as Perflyst opinion.